src/modules/esl-alert/core/esl-alert.ts

```diff
@@ -0,0 +1,85 @@
+import {ExportNs} from '../../esl-utils/environment/export-ns';
+import {bind} from '../../esl-utils/decorators/bind';
+import {attr, jsonAttr} from '../../esl-base-element/core';
+import {ESLBasePopup, PopupActionParams} from '../../esl-base-popup/core';
+import {DeviceDetector} from '../../esl-utils/environment/device-detector';
+
+export interface AlertActionParams extends PopupActionParams {
+  text: string;
+  cls?: string;
+}
+
+@ExportNs('Alert')
+export class ESLAlert extends ESLBasePopup {
+  static is = 'esl-alert';
+  static eventNs = 'esl:alert';
+
+  @jsonAttr<AlertActionParams>({defaultValue: {text: '', cls: ''}})
+  public defaultParams: AlertActionParams;
```

Comment:
Feature Request: There is no possibility for the final consumer to setup default without inheritance to introduce a modified component.
mb let's try to move defaultValue to the static object then.